CMakeLists.txt

```diff
@@ -61,7 +61,9 @@ target_link_libraries(muMatScale PRIVATE m)
 
 target_include_directories(muMatScale PRIVATE ${HDF5_INCLUDE_DIRS})
 
-target_compile_definitions(muMatScale PRIVATE INDEX_SEP)
+if(${BUILD_INDEX_SEP})
```

Comment:
I would add
set(BUILD_INDEX_SEP TRUE CACHE BOOL "Whether to define INDEX_SEP flag")
right before that to keep current default to TRUE